src/Modules/OnPremisesDataGateway/Commands.OnPremisesDataGateway/help/MicrosoftPowerBIMgmt.OnPremisesDataGateway.md

```diff
@@ -0,0 +1,46 @@
+---
+Module Name: MicrosoftPowerBIMgmt.OnPremisesDataGateway
+Module Guid: 
+Download Help Link: 
+Help Version: 
+Locale: 
+---
+
+# MicrosoftPowerBIMgmt.OnPremisesDataGateway Module
+## Description
+Cmdlets to configure on-premises data gateway related items.
+
+## MicrosoftPowerBIMgmt.OnPremisesDataGateway Cmdlets
+### [Add-OnPremisesDataGatewayClusterUser](Add-OnPremisesDataGatewayClusterUser.md)
+Add user and associated permissions to cluster
+
+### [Get-OnPremisesDataGatewayCluster](Get-OnPremisesDataGatewayCluster.md)
+Get clusters for the current user
+
+### [Get-OnPremisesDataGatewayClusterStatus](Get-OnPremisesDataGatewayClusterStatus.md)
+Get cluster status
+
+### [Get-OnPremisesDataGatewayInstaller](Get-OnPremisesDataGatewayInstaller.md)
+Get users authorized to create gateways on the tenant
+
+### [Get-OnPremisesDataGatewayTenantPolicy](Get-OnPremisesDataGatewayTenantPolicy.md)
+Get gateway tenant policy
+
+### [Remove-OnPremisesDataGatewayCluster](Remove-OnPremisesDataGatewayCluster.md)
+Remove a gateway cluster
+
+### [Remove-OnPremisesDataGatewayClusterMember](Remove-OnPremisesDataGatewayClusterMember.md)
```

Comment:
I think the responsibility lies with our team to improve the name. Otherwise, we will have different name for the same concept in only this script. That doesn't make sense to me.